openmdao/core/test/test_residual_sign.py

```diff
@@ -9,10 +9,10 @@
 from openmdao.test.util import assert_rel_error
 
 
-class SimpleImplicit(Component):
+class SimpleImplicitSL(Component):
```

Comment:
actually.. it does not seem to be used in this module at all?  so maybe just move to the modeul it is used in?